doc/source/contribution/documentation/introduction.rst

```diff
@@ -6,6 +6,19 @@ Introduction
 This documentation describes about how to write, generate and manage
 Groonga documentation.
 
+Checkout Groonga from the repository
+------------------------------------
+
+You should build Groonga documentation from source code at the repository.
+Because source code in the repository is the latest.
+
+The Groonga repository is hosted on `GitHub
+<https://github.com/groonga/groonga>`_. Checkout the latest source
+code from the repository::
+
+  % git clone --recursive git@github.com:groonga/groonga.git
+  % cd groonga
```

Comment:
We should use `.. code-block:: console` for console.
See also `doc/source/install/cmake.rst` and so on.

Reply:
fix: [2e847d2](https://github.com/groonga/groonga/pull/1712/commits/2e847d2fdf0e4401e60e2bae7a494c33132ce6e5)